lua/entities/gmod_wire_expression2/core/table.lua

```diff
@@ -37,7 +37,7 @@ local DEFAULT = {n={},ntypes={},s={},stypes={},size=0}
 
 registerType("table", "t", table.Copy(DEFAULT),
 	function(self, input)
-		if IsEmpty(input) then
+		if IsEmpty(input.n) and IsEmpty(input.s) then
```

Comment:
Would `input.size == 0` work? (I see `DEFAULT` has a `size`, but I don't know if it's actually used anywhere.)

Reply:
yep `t.size` is used